Xamarin.Forms.Core/RefreshView.cs

```diff
@@ -53,7 +67,9 @@ public object CommandParameter
 		void RefreshCommandCanExecuteChanged(object sender, EventArgs eventArgs)
 		{
 			if (Command != null)
-				IsEnabled = Command.CanExecute(CommandParameter);
+				SetValueCore(IsEnabledProperty, Command.CanExecute(CommandParameter));
+			else
+				SetValueCore(IsEnabledProperty, true);
```

Comment:
How would the below case work with this change?

```
var refreshView = new RefreshView();
refreshView.IsEnabled = false;
refreshView.Command = new Command(() => {});
```

`RefreshCommandCanExecuteChanged` gets executed and sets `IsEnabled` to true even though I explicitly set it to false.

Reply:
Yea that's an ongoing conversation with Commanding and IsEnabled
https://github.com/xamarin/Xamarin.Forms/issues/2758

So the way it's all done here is just consistent with how it works with Button

```C#
		public static void CommandChanged(IButtonElement sender)
		{
			if (sender.Command != null)
			{
				CommandCanExecuteChanged(sender, EventArgs.Empty);
			}
			else
			{
				sender.IsEnabledCore = true;
			}
		}
```